wpengine.sh

```diff
@@ -162,70 +164,70 @@ fi
 log "Deployment to wpengine $target..."
 
 # clear the tmp folder
-rm -rf ~/deployments/tmp/*
+rm -rf $cwd/tmp/*
```

Comment:
This is a little scary...okay a lot scary :) I'm going to send a pull request your way to address it.

The problem here is not in what you have done in the perfect world.  It is that I follow the mantra of trust no one, and now we have a variable which is the result of a command (pwd) which could be aliased or the result of running the script while in root (silly but possible) and that this could be done by a user with privilege escalated ( either sudo or simply logged in as root ).

The result is that we could have a few things happen...if invoked at root we may see the removal of /tmp causing some system issues.  If invoked elsewhere but pwd is an alias ( either through malicious means or otherwise ), it is unpredicatable....what if say pwd returned "/ /somepath/to/files"  The rm-rf would try to remove the root path and /somepath/to/files.

So we need a little safety before I merge this.  Again I'll send a pull request shortly to your repo with my suggested changes, you can verify and re-add to your patch request.

Thanks!